bbb-common-web/src/main/java/org/bigbluebutton/api/MeetingService.java

```diff
@@ -513,6 +518,7 @@ private void processRemoveEndedMeeting(MeetingEnded message) {
     if (m != null) {
       m.setForciblyEnded(true);
       processRecording(m);
+      if (keepEvents) recordingService.markAsEnded(m.getInternalId());

```

Comment:
@kepstin here is what we discussed about writing `recorded` tag before the `ended` tag. `markAsEnded` will generate the `ended` tag after the `processRecording` method. `processRecording` will call `startIngestAndProcessing` and generate the `recorded` tag. So we should be good to go.

Reply:
Can you just add a comment here to remark why the order is this way? Then everything should be good to go.

Reply:
Sure

Reply:
Done!